src/components/MostRecentScoreboard.tsx

```diff
@@ -28,20 +52,45 @@ function MostRecentScoreboard(props: MostRecentScoresProps) {
   };
 
   return (
-    <div className={styles.grid}>
-      <InfiniteScroll
-        dataLength={items.length} //This is important field to render the next data
-        next={fetchData}
-        hasMore={items.length < plays.length}
-        loader={<LoadingCircle />}
-        endMessage={
-          <p style={{ textAlign: "center" }}>
-            <b>Yay! You have seen it all</b>
-          </p>
-        }
+    <div>
+      <Tabs
```

Comment:
I'd love to keep the Tabs on the screen even if the plays aren't fetched from the API yet. Is it easy? My attempts have failed:

https://github.com/greg-finley/alphabet-game-app/commit/c6549f2aae1fed1043c4cbbdd160bf8891a75a02

https://github.com/greg-finley/alphabet-game-app/commit/a3ae5f7d8f159eb642d7f83eab53fdfb9fae11cd